extensions/ql-vscode/src/view/model-editor/ModeledMethodDataGrid.tsx

```diff
@@ -84,6 +91,9 @@ export const ModeledMethodDataGrid = ({
             <VSCodeDataGridCell cellType="columnheader" gridColumn={5}>
               Kind
             </VSCodeDataGridCell>
+            {viewState.showMultipleModels && (
+              <VSCodeDataGridCell cellType="columnheader" gridColumn={6} />
```

Comment:
Optional: It would be nice for accessibility if we could add some text here that's only visible for screen readers (like using the example `.sr-only` CSS shown [here](https://webaim.org/techniques/css/invisiblecontent/).

Reply:
Good idea. I've added some table header content and a new `ScreenReaderOnly` styled component. What do you think?

Reply:
Thanks for adding that, I think adding a new component is very useful if we ever want to re-use this anywhere else.